documentation/src/main/wiki/PerformanceTestsMethodology.md

```diff
@@ -0,0 +1,95 @@
+# Performance Tests Methodology
+
+## Introduction
+This page describes methodology of Knotx application performance tests.
+
+## Goals
+Goals of conducted tests are as following:
```

Comment:
This sentence is incorrect. Please change it to the following one:

_**The** goals of conducted tests are **as follows**:_